documents/charter.md

```diff
@@ -1,14 +1,20 @@
 
 Video traffic usually employs adaptive bitrate (ABR) schemes to dynamically adjust the video quality (and thus the data rate) in response to changing network conditions. In the presence of traffic shaping, the ABR scheme should ideally adapt the quality and converge on a bitrate sustainable by the shaper. In practice this is extremely difficult to achieve while maintaining a good user experience. Application providers are even designing algorithms to detect the presence of such traffic shapers and estimate the targeted shaping rate, however, these algorithms are inaccurate and complex. Instead, it would be beneficial, for both the application provider and network operator, to signal the shaper rate to the application to self-adapt their video traffic to conform to the specified characteristics. Application provider has the ability to measure end user QoE and therefore can self-adapt with QoE feedback.
 
-The Secure Communication of Network Properties (SCONE-PRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as maximum achievable bandwidth for a video. The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video using QUIC connections carrying those applications. Further use cases may be considered later in the working group, however, it is not assumpted that all signaling use cases must or can be addressed by by the same protocol.
+The Secure Communication of Network Properties (SCONEPRO) Working Group's primary objective is to specify an on-path protocol for securely communicating network properties to clients relevant to a given application, such as maximum achievable bandwidth for a video. The working group will initially focus on a solution that communicates the maximum achievable bandwidth for a video using QUIC connections carrying those applications. Further use cases may be considered later in the working group, however, it is not assumpted that all signaling use cases must or can be addressed by by the same protocol.
 
 The properies of this mechanism are as follows:
 
 1. On-path establishment. That is, no off-path element is needed to establish the communication channel between the entity communicating the properies and the client.
 2. Security. The mechanism must ensure the confidentiality, integrity, and authenticity of the communication. The mechanism must have independent security context from the application's security context. The group must not define new security mechanisms for this purpose.
 3. Associativity with an application. The network properties must be associated with a given application traversing the network, for example a video playback.
 4. Scalability. The mechanism must be scalable and implementable by Internet infrastructure as it exists today, for example mobile network packet cores.
+
+Milestones
```

Comment:
I would like us to go a bit verbose on the items here and rename this as deliverables.

Example - the first two sub bullets can be elaborated as -

- Develop a standard track "SCONEPRO protocol" to securely communicate network information to clients
   - specify protocol requirements for the base "SCONEPRO protocols."
   - specify connection establishment, communication signaling and data format (container to share network information)
   - specify discovery mechanism (if required)
   - specify and justify privacy and security requirements
   
- Develop a standard track specification based on SCONEPRO protocol to communicate "maximum available bandwidth" to the client(s)
    - describe use case and motivation
    - specify exact encoding of "maximum available bandwidth" in the SCONEPRO protocol container

I think this we will be very clear on what the deliverable means and reflect how well we are thinking about the job at hand. I hope this make sense. As I said the above is just example details the proponents must have more meaningful terms and detailed description.

Reply:
Zahed, 

> I would like us to go a bit verbose on the items here and rename this as deliverables.
> 
> Example - the first two sub bullets can be elaborated as -
> 
> * Develop a standard track "SCONEPRO protocol" to securely communicate network information to clients
>   
>   * specify protocol requirements for the base "SCONEPRO protocols."
>   * specify connection establishment, communication signaling and data format (container to share network information)
>   * specify discovery mechanism (if required)
>   * specify and justify privacy and security requirements
> * Develop a standard track specification based on SCONEPRO protocol to communicate "maximum available bandwidth" to the client(s)
>   
>   * describe use case and motivation
>   * specify exact encoding of "maximum available bandwidth" in the SCONEPRO protocol container
> 
> I think this we will be very clear on what the deliverable means and reflect how well we are thinking about the job at hand. I hope this make sense. As I said the above is just example details the proponents must have more meaningful terms and detailed description.

This does make sense to me. I'll wait until I can touch base with the other proponents on the details. 

Thank you for your suggestion!

Reply:
I think this makes sense. 
The only thing I'm (a bit) worried about is if this can become too restrictive. 
For instance, we might want to specify the SCONEPRO information elements and data format independently from the SCONEPRO protocol.